<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextParam</class>
 <widget class="QWidget" name="ContextParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Context Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="safetyDepth">
       <property name="toolTip">
        <string>Selected safety depth</string>
       </property>
       <property name="text">
        <string>Safety depth</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="shallowContour">
       <property name="toolTip">
        <string>Selected shallow water contour</string>
       </property>
       <property name="text">
        <string>Shallow contour</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="twoShades">
       <property name="toolTip">
        <string>Show two depth shades only, safe vs unsafe</string>
       </property>
       <property name="text">
        <string>Two shades</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="twoShadesValue">
       <property name="text">
        <string>Safe</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="deepContour">
       <property name="toolTip">
        <string>Selected deep water contour</string>
       </property>
       <property name="text">
        <string>Deep contour</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="shallowPattern">
       <property name="toolTip">
        <string>Show the shallow diamond pattern in shallow water</string>
       </property>
       <property name="text">
        <string>Shallow pattern</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="showDangersValue">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="plainBoundaries">
       <property name="toolTip">
        <string>Simplify portrayal of line</string>
       </property>
       <property name="text">
        <string>Plain boundaries</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="simplifiedPoints">
       <property name="toolTip">
        <string>Simplify portrayal of point symbols</string>
       </property>
       <property name="text">
        <string>Simplified points</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="simplifiedPointsValue">
       <property name="text">
        <string>Simplify</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="radarOverlay">
       <property name="toolTip">
        <string>Tells the portrayal whether a RADAR sweep is displayed or not</string>
       </property>
       <property name="text">
        <string>Radar overlay</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="radarOverlayValue">
       <property name="text">
        <string>Displayed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="ignoreScamin">
       <property name="toolTip">
        <string>Ignore the scaleMinimum attribute</string>
       </property>
       <property name="text">
        <string>Ignore scamin</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="ignoreScaminValue">
       <property name="text">
        <string>Ignore</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="nationalLanguage">
       <property name="toolTip">
        <string>Selects which language to use for text (ISO 639-3)</string>
       </property>
       <property name="text">
        <string>National language</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="nationalLanguageValue">
       <item>
        <property name="text">
         <string>Eng</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="safetyContour">
       <property name="toolTip">
        <string>Selected safety contour</string>
       </property>
       <property name="text">
        <string>Safety contour</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="shallowPatternValue">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="safetyDepthValue">
       <property name="suffix">
        <string> m.</string>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="plainBoundariesValue">
       <property name="text">
        <string>Simplify</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="shallowContourValue">
       <property name="suffix">
        <string> m.</string>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="deepContourValue">
       <property name="suffix">
        <string> m.</string>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="showDangers">
       <property name="toolTip">
        <string>Show isolated dangers in shallow waters</string>
       </property>
       <property name="text">
        <string>Show dangers</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="safetyContourValue">
       <property name="suffix">
        <string> m.</string>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
